src/main/java/edu/brown/cs/student/main/Main.java

```diff
@@ -73,33 +76,175 @@ private void run() {
       runSparkServer((int) options.valueOf("port"));
     }
 
-    // TODO: Add your REPL here!
-    MathBot math = new MathBot();
+    // initialize a StarDistance object
+    StarDistance distCalculator = new StarDistance();
+
     try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
       String input;
+      // read the INPUT by line
       while ((input = br.readLine()) != null) {
         try {
           input = input.trim();
-          String[] arguments = input.split(" ");
-          // TODO: complete your REPL by adding commands for addition "add" and subtraction
-          //  "subtract"
-          if (arguments.length == 3 && this.isNumeric(arguments[1]) && this.isNumeric(arguments[2])) { // && arguments[1] & arguments[2] are both numeric)
```

Comment:
The addition of an isNumeric method was a smart decision. This eliminates the problem of encountering issues when using double.parseDouble with a non numeric value.